rfcs/2023-12-07-config-options.md

```diff
@@ -0,0 +1,232 @@
+# Configuration Options
+
+| Feature name  | Config Options                               |
+| :------------ | :------------------------------------------ |
+| **Author(s)** | Nick Tindle (<nicholas.tindle@agpt.co>), J. Zane Cook (<jzanecook@z90.studio>)                      |
+| **Relevant Issue:**   | [#39](https://github.com/AI-Engineer-Foundation/agent-protocol/issues/39)                                 |
+| **RFC PR:**   | [TBD](https://github.com/AI-Engineer-Foundation/agent-protocol/pulls)                                 |
+| **Created**   | 2023-12-07                                  |
+| **Updated**   | 2023-12-15                                  |
+
+## Summary
+
+We would like to add support for `config_options` to relace most usages of `addtional_input` so that we can provide clear and structured configrations to our agents. This will fix the "unknown unknowns" problem that exsits with many agents today of not knowing what the valid additional inputs are for a given representation of an agent.
```

Comment:
Can you be clear that the `config_options` is done to specify the schema of `additional_input`